mud-contracts/world/src/codegen/common.sol

```diff
@@ -15,3 +15,8 @@ enum SmartAssemblyType {
   SMART_TURRET,
   SMART_GATE
 }
+
+enum KillMailLossType {
+  SHIP,
+  POD
+}
```

Comment:
can you override the enum values in solidity? else we should maybe add the "UNSPECIFIED" type to match the proto enum.

```suggestion
enum KillMailLossType {
  UNSPECIFIED,
  SHIP,
  POD
}
```

the enum in protobuf: 

```proto
enum LossType {
  LOSS_UNSPECIFIED = 0;
  SHIP = 1;
  POD = 2;
}
```

Reply:
Yes I did think about that.

The unspecified part of the proto is there because proto "requires" it.

We should never store kill mails that have an unspecified loss type.

Reply:
yeah and we never will, but it was mostly to have the same enum value everywhere. 
Else we'll have `SHIP=0` in indexer/chain and `SHIP=1` in game/protobuf.

if we can do stuff like that it also works: 
```suggestion
enum KillMailLossType {
  SHIP=1,
  POD=2
}
```

Just a nitpick anyway 👍

Reply:
I understand the reasoning.

I think we should do a mapping in the code the reads from the contract. The reason for that is that if the enum order is changed in one place but not the other (someone might forget to do it in both places, might not know about the proto definitions, etc.), then it will break the system.